myapp/app/controllers/application_controller.rb

```diff
@@ -1,2 +1,26 @@
 class ApplicationController < ActionController::Base
+
+  rescue_from Exception,                      with: :_render_500
+  rescue_from ActiveRecord::RecordNotFound,   with: :_render_404
+  rescue_from ActionController::RoutingError, with: :_render_404
+
+  def routing_error
+    raise ActionController::RoutingError, params[:path]
+  end
+
+  private
+
+    def _render_404(e = nil)
+      if e.present?
+        logger.error "Rendering 404 with exception: #{e.message}"
+      end
```

Comment:
```suggestion
      logger.error "Rendering 404 with exception: #{e.message}" if e.present?
```
レビュー指摘前のようにif文を後置で一行にしてあげることがRubyではできます。
こちらの方が無駄に行数が増えない、また１行でシンプル且つRubyらしい記述となります。
javaでは上記が正しいですが、Rubyでは１行にすることが思想としてるRubyらしさになるため、慣れてもらう意味でも意識して記載してもらえると嬉しいです。

Reply:
１行で記載するよう修正しました。

Rubyでは１行で記載する思想とのこと、承知しました。
今後意識して記載していきます。

Reply:
まだ修正されてないようですが、コミットプッシュ漏れですかね？

Reply:
失礼しました。コミットが漏れておりました。
プッシュまで行い、ソースが修正されていることを確認いたしました。

Reply:
修正対応ありがとうございます！